package/MDAnalysis/lib/util.py

```diff
@@ -295,7 +295,7 @@ def anyopen(datasource, mode='r', reset=True):
                     stream.reset()
                 except (AttributeError, IOError):
                     try:
-                        stream.seek(0L)
```

Comment:
https://github.com/MDAnalysis/mdanalysis/blob/develop/package/MDAnalysis/coordinates/TRZ.py#L351

I think I'm thinking of this in TRZ, where I got a bug in larger files (when the size in bytes caused an int overflow)

I opted not to include the ~4gb file that requires long ints when seeking :D